build/.gitignore

```diff
@@ -0,0 +1,3 @@
+# ignore vscode files (debug config etc...)
```

Comment:
it looks like this `build` folder is included here directly as a directory in your repository, but I believe it's supposed to be a submodule instead that points at https://github.com/upbound/build/ so your repo can easily pick up fixes and new functionality from there.

see in the [provider-template repo](https://github.com/crossplane/provider-template/) how this is done.

Reply:
Thanks , I didn't notice it :)